docs/user-manual/consume.md

```diff
@@ -5,25 +5,78 @@ slug: /consume
 ---
 
 # Consume fact statements
+
 A fact statement is a datum that the Orcfax network publishes to a Cardano
-blockchain transaction. This data point becomes an input to Cardano smart
+blockchain transaction. This This datum can then be used as an input to Cardano smart
 contracts and dApps.
 
-All fact statements are structured as [JSON-LD](https://json-ld.org/) files
+## UTxO model
+
+A single fact statement is associated with a single [UTxO][utxo-1]. Each UTxO
+has a [reference input][reference-1] encoded as an [inline datum][datum-1].
+
+Reference inputs are published using a COOP authentication token with a COOP
+minting policy identifier, currently:
+
+* preprod: `5ec8416ecd8af5fe338068b2aee00a028dc1f4c0cd5978fb86d7c038`
+  (smart contract address:
+  `addr_test1wrtcecfy7np3sduzn99ffuv8qx2sa8v977l0xql8ca7lgkgmktuc0`)
+* mainnet: `84da51fd8041b6a1073f1154ddde0c846d71c4081fc1f7ba66e0e6b5`
+  (smart contract address:
+  `addr1w8tcecfy7np3sduzn99ffuv8qx2sa8v977l0xql8ca7lgkgq7lqh2`)
+
+Minting policy identifiers are critical as fact statements should be verified as
+coming from a known source. These identifiers also provides the ability to
+filter all of the UTxO at a given smart contract address and to filter out
+those sent by others to the same address.
+
+Users will therefore need to use the following information to identify the
+latest facts.
+
+* Minting policy ID.
+* Datum format, including:
+  * Most recent (`ValueReference -> PropertyValue[1] -> value)`, i.e. the
+  largest POSIX timestamp compared to other fact statement datum,
+  * Feed name, e.g. "ADA-USD".
+
+> Note: Users on preprod will find a volume of test data that hasn't been
+> retired as Orfax previously used this space as a testing sandbox. Users can
+> ignore this data by using the latest preprod policy ID and fact schema.
+
+Unconsumed reference inputs are those still available for smart contracts to use
+on-chain. Spending a UTxO with a reference input renders it unusable in a smart
+contract (though the data will be visible in the historical transaction).
+Consuming reference inputs is sometimes also referred to as garbage collection
+or *retiring* of fact statements. Orcfax's mainnet V1 will operate a policy
+whereby two unspent fact UTxO will remain on-chain at all times.
```

Comment:
Do we need to explain why this policy has been enacted?
"or *retiring* of fact statements. Orcfax's mainnet V1 will operate a policy
whereby two unspent fact UTxO will remain on-chain at all times. In the
case of publication (or Cardano block) delays, this policy will work to
ensure that smart contracts have access to on-chain reference inputs."
?